src/openclsim/core/processor.py

```diff
@@ -40,10 +40,8 @@ def process(
         Yields the time it takes to process.
         """
 
-        assert isinstance(origin, HasContainer) or isinstance(origin, HasContainer)
-        assert isinstance(destination, HasContainer) or isinstance(
-            destination, HasContainer
-        )
+        assert isinstance(origin, HasContainer) # ?? or isinstance(origin, HasContainer)
```

Comment:
```suggestion
        assert isinstance(origin, HasContainer)
```

Reply:
it had two isinstance calls, I do not know why, I think 1 is ok. I kept this comment as a reminder for something I might not know